.github/workflows/continuous-integration.yml

```diff
@@ -374,9 +375,10 @@ jobs:
         with:
           name: cypress-mochawesome-test-results-artifact-${{ matrix.ci_node_index }}
           path: cypress/results
+          retention-days: 1
```

Comment:
@holdenhinkle: Just confirming that cleaning up the Mochawesome artifacts after 1 day is okay. I figured they would've served their purpose by the time they're gotten reported to BigQuery.

Reply:
Yes, it's totally okay to clean them up.